setup_app.py

```diff
@@ -0,0 +1,36 @@
+from flask import Flask
+
+from util.error import UNKNOWN_RESSOURCE
+
+
+def setup_app(app_config):
+    """Create an app using the given config.
+
+    Arguments:
```

Comment:
Maybe change comments to [docstring format](https://stackoverflow.com/questions/3898572/what-is-the-standard-python-docstring-format) as discussed?